content/applications/marketing/surveys/overview/time_random.rst

```diff
@@ -1,34 +1,46 @@
-============================
-Time and Randomize Questions
-============================
+==============================
+Timed and Randomized Questions
+==============================
 
 Time Limit
 ==========
 
-| On timed surveys, respondents need to complete the survey within a certain period of time. It can
-  be used to ensure that all respondents get the same amount of time to find the answers, or to
-  decrease the chance of having them looking at external resources.
-| Set the *Time limit* under the tab *Options*.
+During a timed survey, participants must finish the survey within a certain period of time. Placing
+a time limit on your survey ensures that every participant gets the same amount of time to fill in
+answers. It also greatly reduces the chance of participants looking up responses, via external
+resources (Google, etc.).
 
-.. image:: time_random/tab_options_timelimit.png
+The :guilabel:`Time Limit` field is found under the :guilabel:`Options` tab of the
+:guilabel:`Survey Form` under the :guilabel:`Time and Scoring` section.
+
+.. image:: time_random/time-limit.png
```

Comment:
This screenshot should be retaken. I think the screenshot shows Odoo 15's interface, not Odoo 14, which is the target branch of this PR. Don't forget to compress your image file size as well with pngquant!

Before making changes to the RST:

1. Retake the screenshot (remember to zoom in 110-125% and make sure the final image width/length is less than 768px)
2. Compress the image file with pngquant
3. Save your compressed image in an accessible place

During making changes to RST (using the 'Follow these steps to make another commit on an existing PR (squash commits)' directions):

1. Do Steps 1-6 of 'Follow these steps to make another commit on an existing PR (squash commits)' directions
2. Copy the file path of the original image frontend-badges.png in the scoring image folder
3. Run `git rm [paste-file-path]` on your terminal
4. Cut the new, compressed image file and paste it into the scoring image folder
5. Continue from Step 7 of 'Follow these steps to make another commit on an existing PR (squash commits)' directions